util/reggen/uvm_reg_base.sv.tpl

```diff
@@ -326,7 +326,11 @@ reg, mr, reg_idx)">\
 <%
     if compact_field_inst_name:
       reg_field_name = compact_field_inst_name
-      if len(fields) > 1:
+      # If len(fields) > 1, we define fields as an array and we need an indice to refer to the
```

Comment:
The first time I read it, I got `fields` confused with the Python data structure `fields` use in this loop, which is already an array. I suppose you mean UVM fields in this statement here?

Reply:
yes, update wording to make it clear